README.md

```diff
@@ -1 +1,59 @@
-# ruby-enumerables
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Ruby Enumerables
+> A recreation of the #any?, #all? and #filter Enumerable methods. Built with Ruby.
+
+
+
+## Built With
+
+- Ruby
+
+## Requirements 
+Install:
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+- [IRB](https://github.com/ruby/irb)
+
+## Getting Started
+
+To get a local copy follow these simple example steps.  
+
+1.- Open the terminal window and clone the repository using this command:  
+`git@github.com:leopiresgoss/ruby-enumerables.git` 
+
+2.- Change the directory to the react-bookstore directory  
+`cd ruby-enumerables`  
+3.- Install the project's dependencies by running this command:   
+`bundle install`   
+4.- In you terminal, run `irb`, following this sequence: copy the enumerable.rb and my_list.rb code and paste there, then you can check the results  
+
+## Authors
+👤 **Hemerson Foreste**
+- GitHub: [ForHemer](https://github.com/ForHemer)
+- Twitter: [@HemersonForeste](https://twitter.com/HemersonForeste)
+- [LinkedIn](https://linkedin.com/in/hemerson-foreste-890685197)
+
+👤 **Leonardo Pires Goss**
+- GitHub: [leopiresgoss](https://github.com/leopiresgoss)
+- Twitter: [@leonardopgoss](https://twitter.com/leonardopgoss)
+- [LinkedIn](https://www.linkedin.com/in/leonardogoss/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/leopiresgoss/space-travelers/issues).
+
```

Comment:
- [ ] Kindly fix this link as it is different from the `enumerable` project issues. By fixing this error your project will be ready to be submitted.